### ISSUE TITLE
specify rwd_agg by reading in a csv file?

### ISSUE THREAD

**Issue by rabutler** [BoulderCodeHub/RWDataPlyr#70]
Right now, it's two steps, but we could make it part of the constructor?

Right now: 
```
ra <- rwd_agg(read.csv("file", stringsAsFactors = FALSE))
```

Could do:
```
ra <- rwd_agg(file = "file.csv")
```
Or `read_rwd_agg()` might be better approach.

**Comment by rabutler** [comment]
going with `read_rwd_agg()`

**Comment by rabutler-usbr** [comment]
closed by #92
